products/composer.md

```diff
@@ -6,9 +6,16 @@ changelogTemplate: "https://getcomposer.org/changelog/__LATEST__"
 auto:
 -   git: https://github.com/composer/composer.git
 releases:
--   releaseCycle: "2.3"
+-   releaseCycle: "2.4"
     eol: false
     support: true
+    latest: "2.4.1"
+    latestReleaseDate: 2022-08-20
+    releaseDate: 2022-08-16
+
+-   releaseCycle: "2.3"
+    eol: true
```

Comment:
if this became eol with release of 2.4 then it might be much better to put
date of 2.4 series release instead of using directly true 
i mean eol: 2022-08-16